### ISSUE TITLE
[dqm] Notify Data Quality Validation Results to Slack

### ISSUE THREAD

**Issue by TremaMiguel** [feast-dev/feast#2613]
**Is your feature request related to a problem? Please describe.**

Is possible to send a Slack Notification that includes information about Data Validation to be aware of issues arising on new available data.

New datasets being validated using the `GE Profiler` could notify to Slack any expectations not met. For example, an additional parameter could be passed to the function `to_df`

```python
# Example taken from docs: https://docs.feast.dev/v/master/tutorials/validating-historical-features#5.-validating-new-historical-retrieval
job.to_df(
     validation_reference=validation_reference,
     notify_slack=True
)
```  

Great Expectations already support an [SlackValidationAction](https://legacy.docs.greatexpectations.io/en/latest/guides/how_to_guides/validation/how_to_trigger_slack_notifications_as_a_validation_action.html) that summarizes the expectations that new data satisfies:

![Captura de pantalla (1167)](https://user-images.githubusercontent.com/49818188/165207389-b19e6409-fd14-499d-b6e4-9b5d33497616.png)

Alternatively a custom implementation that sends a request to an Slack Webhook is an option.

**Comment by woop** [comment]
Hey @TremaMiguel, thanks for the suggestion. It makes sense to me. One question: Wouldn't it be more convenient and expected to enable slack configurations in the `feature_store.yaml` or the feature repo, as opposed to the `to_df()` call?

**Comment by TremaMiguel** [comment]
yes, I think it can be part of a `data_quality_monitor` configuration 
```
project: loyal_spider
registry: data/registry.db
provider: local
data_quality_monitor:
    slack_notification:
        slack_weekhook: ${slack_webhook}
        other_param: 'other_param'
```

I think this makes sense more if user wants to notify all kind of results from expectation suite `failure` or `success` from all registered Feature Groups.

Or if he wants to notify expectations results from specific feature groups he can use the SDK. 

What are your thoughts?
